From akpm@osdl.org Fri Jan 13 17:20:04 2006
Message-Id: <200601120407.k0C477Fi028701@shell0.pdx.osdl.net>
Subject: + tulip-remove-duplicate-pci-ids.patch added to -mm tree
To: jurij@wooyd.org, maks@sternwelten.at, mm-commits@vger.kernel.org
From: akpm@osdl.org
Date: Wed, 11 Jan 2006 20:06:54 -0800

tulip driver advertises support for non-working id
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=334104

It looks like tulip and dmfe have some identical PCI IDs in their lists of
supported devices, which, I believe, should not normally happen.  Attached
patch removes the PCI IDs handled by dmfe driver from tulip.

Signed-off-by: Jurij Smakov <jurij@wooyd.org>
Signed-off-by: maximilian attems <maks@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 drivers/net/tulip/tulip_core.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN drivers/net/tulip/tulip_core.c~tulip-remove-duplicate-pci-ids drivers/net/tulip/tulip_core.c
--- devel/drivers/net/tulip/tulip_core.c~tulip-remove-duplicate-pci-ids	2006-01-11 20:06:50.000000000 -0800
+++ devel-akpm/drivers/net/tulip/tulip_core.c	2006-01-11 20:06:50.000000000 -0800
@@ -223,8 +223,6 @@ static struct pci_device_id tulip_pci_tb
 	{ 0x1259, 0xa120, PCI_ANY_ID, PCI_ANY_ID, 0, 0, COMET },
 	{ 0x11F6, 0x9881, PCI_ANY_ID, PCI_ANY_ID, 0, 0, COMPEX9881 },
 	{ 0x8086, 0x0039, PCI_ANY_ID, PCI_ANY_ID, 0, 0, I21145 },
-	{ 0x1282, 0x9100, PCI_ANY_ID, PCI_ANY_ID, 0, 0, DM910X },
-	{ 0x1282, 0x9102, PCI_ANY_ID, PCI_ANY_ID, 0, 0, DM910X },
 	{ 0x1113, 0x1216, PCI_ANY_ID, PCI_ANY_ID, 0, 0, COMET },
 	{ 0x1113, 0x1217, PCI_ANY_ID, PCI_ANY_ID, 0, 0, MX98715 },
 	{ 0x1113, 0x9511, PCI_ANY_ID, PCI_ANY_ID, 0, 0, COMET },
_


